xarray/tests/test_plot.py

```diff
@@ -65,7 +70,7 @@ def easy_array(shape, start=0, stop=1):
 
 
 @requires_matplotlib
-class PlotTestCase(TestCase):
+class PlotTestCase(unittest.TestCase):
```

Comment:
I thought we were replacing the last of `unittest` here?

No worries on saving this for later, though :)